src/main/flight/imu.c

```diff
@@ -263,7 +263,7 @@ static void imuMahonyAHRSupdate(float dt, float gx, float gy, float gz,
     // Use measured acceleration vector
     float recipAccNorm = sq(ax) + sq(ay) + sq(az);
     if (useAcc && recipAccNorm > 0.01f) {
-        // Normalise accelerometer measurement
+        // Normalise accelerometer measurement when smoothed acc is within 20% of 1G
```

Comment:
20% margin is tested elsewhere. This only prevents division by zero.

Reply:
True, but useAcc is only true when all smoothed axes are within 20% of 1G.
I'll modify the comment, but I wanted it to let the reader know that the code would not run unless the smoothed acc value on every axis is within 20% of 1G, ie the quad is stable.